app/lib/three_scale/bot_protection/form.rb

```diff
@@ -6,6 +6,10 @@ module Form
       include Base
       include Recaptcha::Adapters::ViewMethods
 
+      # This module must be included from form builders or any other
+      # form-related class that responds to `:template`
```

Comment:
It seems like this comment is supposed to reside at line 5 if I'm not mistaken.

Reply:
I think it's OK. I placed it here because it's accessing `:template` at the line below.

Reply:
The module is not required to be included in all classes that implement `:template`, but if you include it in a class, that class must implement `:template`

Reply:
> ... can only be included ...

but then this is obvious when you see the delegate. So the comment would be redundant.

Reply:
removed